jobs/video_jobs/maya_submit.sh.erb

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+#set the resources requests. Job name, time, nodes and cores per task, task count 
+
+#PBS -l nodes=1:ppn=<%= cores %>
+#PBS -j oe
+#PBS -o <%= base_output_dir.to_s %>/$PBS_JOBID/$PBS_JOBID.out
```

Comment:
You should be able to just do:

```
#PBS -o batch_jobs/$PBS_JOBID/$PBS_JOBID.out
```

instead of having the absolute path embedded. From the manpage on -o option of qsub:

>Where path_name is not an absolute path name, then the qsub command will expand the path name relative to the current working directory of the command. The command will supply the name of the host upon which it is executing for the hostname component.